GiEnJul/ClientApp/src/components/admin/overview/OverviewMacro.tsx

```diff
@@ -0,0 +1,191 @@
+import React, { useEffect, useState } from "react";
+import Giver from "./Giver";
+import Recipient from "./Recipient";
+import { Button, Grid, Typography } from "@material-ui/core";
+import usePost from "../../../hooks/usePost";
+
+export interface SelectedConnectionType {
+  giverRowKey: string;
+  giverPartitionKey: string;
+  recipientRowKey: string;
+  recipientPartitionKey: string;
+}
+
+export interface GiverType {
```

Comment:
Would it be nice to have the `GiverType`, `RecipientType` and `PersonType` in a `types.tsx` (or something) file?

Reply:
agree